### PR TITLE
Linking ToDo-List to Avalonia.Samples

### PR DESCRIPTION
**Pull request by timunie**
what will we do with the i18n docs? I'm not planning to translate Avalonia.samples right now. Let me know your thoughts.

### CODE REVIEW

**Comment by timunie** [comment]
@stevemonaco please wait until merge as I need to check localized docs and images first. Thx 🙏

**Review by Arhell — APPROVED** [review]

**Comment by stevemonaco** [comment]
The new changes LGTM. 👍

Anything else can probably wait for user feedback and localizers. Maybe after initial feedback, localizers can submit a ru / zhHans .adoc to Samples where it will live.

**Comment by timunie** [comment]
> Maybe after initial feedback, localizers can submit a ru / zhHans .adoc to Samples where it will live. 

probably, yes. Thx ❤️

**Pull request merged**
